dataactvalidator/config/sqlrules/a10_appropriations.sql

```diff
@@ -1,8 +1,14 @@
+WITH appropriation_a10 AS 
```

Comment:
I tested the `WITH appropriation_a10` won't cause any issues when 2 submission are running the same rule at the same time. I can also add the `[submission-id]` to be extra careful?